src/main/java/org/killbill/billing/plugin/catalog/test/CatalogTestPluginApi.java

```diff
@@ -103,30 +80,19 @@ public DateTime getLatestCatalogVersion(Iterable<PluginProperty> properties, Ten
 
     @Override
     public VersionedPluginCatalog getVersionedPluginCatalog(Iterable<PluginProperty> properties,
-                                                            TenantContext tenantContext) {
+            TenantContext tenantContext) {
 
         System.err.println("++++++++++++  FOUND TENANT " + tenantContext.getTenantId() + ", accountId = " + tenantContext
                 .getAccountId());
-        final VersionedPluginCatalog result = new VersionedPluginCatalogModel(catalogName,
-                                                                              toStandalonePluginCatalogs(versions));
-        logger.info("CatalogTestPluginApi getVersionedPluginCatalog returns result.. ");
+
+        VersionedPluginCatalog result = readCatalogFromResource();
+
+        if(result == null) {
```

Comment:
I think it would be best to throw an exception in that case - but not super important at this stage.